src/Raven.Studio/package.json

```diff
@@ -103,6 +103,7 @@
     },
     "dependencies": {
         "@hookform/resolvers": "^3.9.0",
+        "@juggle/resize-observer": "^3.4.0",
```

Comment:
Avoid using an external library if possible

Reply:
It should be in dev dependency?

Reply:
rather no